packages/react/test/specs/lib/felaRenderer-test.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { createSnapshot } from 'jest-react-fela'
+import { EmptyThemeProvider } from 'test/utils'
+import Box from 'src/components/Box/Box'
+import Animation from 'src/components/Animation/Animation'
+import Icon from 'src/components/Icon/Icon'
+import Provider from 'src/components/Provider/Provider'
+import Text from 'src/components/Text/Text'
+import { felaRenderer } from 'src/lib'
+import { teams } from 'src/themes'
+
+test('css fallback value', () => {
+  const snapshot = createSnapshot(
+    <EmptyThemeProvider>
+      <Box styles={{ color: 'red' }} />
+    </EmptyThemeProvider>,
+    {},
+    felaRenderer,
+  )
+  expect(snapshot).toMatchSnapshot()
+})
+
+test('keyframe', () => {
+  const spinner = {
+    keyframe: ({ fromColor, toColor }) => ({
+      from: {
+        color: fromColor,
+      },
+      to: {
+        color: toColor,
+      },
+    }),
+    keyframeParams: {
+      fromColor: 'red',
+      toColor: 'blue',
+    },
+    duration: '5s',
+  }
+
+  const snapshot = createSnapshot(
+    <Provider theme={teams}>
```

Comment:
This adds insane amount of styles to the snapshot (fonts, static styles).
It makes the snapshot difficult to review and the test fragile.
Can we do it with just inline minimalistic theme?

Reply:
If I do not wrap it in Teams theme, then the resulting snapshot doesn't contain the `keyframe` section. How can I create such a minimal theme, that will allow me to use the Animation's `keyframe`s?